packages/cli/src/pm2-async.js

```diff
@@ -0,0 +1,10 @@
+const pify = require('pify')
+const pm2 = require('pm2')
+
+module.exports.pm2Connect = pify(pm2.connect.bind(pm2))
```

Comment:
Why not to use native `util.promisify` instead of a new dep?

Reply:
Solved. Thanks.